src/targets/gpu/kernels/include/migraphx/kernels/types.hpp

```diff
@@ -74,8 +74,10 @@ static_assert(sizeof(uint64_t) == 8, "uint64_t must be 8 bytes");
 template <class T, index_int N>
 using vec = T __attribute__((ext_vector_type(N)));
 
+using bf16 = __hip_uint16_t;
 using half  = _Float16;
 using half2 = migraphx::vec<half, 2>;
+using bf16 =  ushort;
```

Comment:
There is duplicate typedefs here and they are both wrong. It should be `using bf16 = __hip_bfloat16`.